### PR TITLE
Cheatsheets (CLI and VM) & TOC update

### LINKED ISSUE #111: CLI Cheatsheet
Short and efficient page providing commands, descriptions and examples of common tasks!
@jmorel, @samlesu, @Esadruhn I don't remember if something specific was agreed about this, but I'd be happy to start and to get your feedbacks!

### PR DESCRIPTION
**Pull request by natct10**
- [x] cli cheatsheet
- [x] vm cheatsheet
- [x] Toc updates

close #111 & #120

### CODE REVIEW

File: src/cheatsheet_vm.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Quick deploy on Ubuntu 20.04 VM
+
+> /!\ This configuration is totally not suitable for production environment!
+
+```sh
+# Check the machine
+uname -a
```
Comment (bowni) [inline src/cheatsheet_vm.md:7]: Not a bad idea to start with this, but the question is: "so what?" 😄 
Maybe add a cmoment that if the system replies "Windows" of some sort then the user should be worried with the rest... :)

**Review by bowni — COMMENTED** [review]
Very cool! In the `cheatsheet_vm.md` I would suggest to put more section titles, structure a bit more the content maybe, what do you think?

File: src/cheatsheet_vm.md (hunk 1 above)
Comment (natct10) [reply src/cheatsheet_vm.md:7]: Thank you @bowni for the feedbacks! I have updated the document accordingly!

File: src/cheatsheet_cli.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- dataset
+- objective
+
+## Run-local
+
+TODO
+
+## Tips
+
+By default, `--pretty` formatting is used, but you might want to use a yaml or json output format with `-o` for `--output` with the arguments `yaml` or `json`.
```
Comment (jmorel) [inline src/cheatsheet_cli.md:106]: There is no `--pretty` option, it's all `-o pretty` now

Comment (natct10) [reply src/cheatsheet_cli.md:106]: Oh thank you @jmorel, I will update this!

**Comment by natct10** [comment]
The TOC is updated as follow: image

**Pull request merged**